pyreal/explainers/base.py

```diff
@@ -281,21 +256,21 @@ def model_predict(self, x_orig):
         x_model = self.transform_to_x_model(x_orig)
         return self.model.predict(x_model)
 
-    def model_predict_on_explain(self, x_explain):
+    def model_predict_on_algorithm(self, x_algorithm):
```

Comment:
In what case do we need this function? I think we could always call the model on the original feature space.